src/pages/EditReportFieldDropdownPage.tsx

```diff
@@ -0,0 +1,82 @@
+import React, {useMemo, useState} from 'react';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import OptionsSelector from '@components/OptionsSelector';
+import ScreenWrapper from '@components/ScreenWrapper';
+import useLocalize from '@hooks/useLocalize';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useThemeStyles from '@hooks/useThemeStyles';
+
+type EditReportFieldDropdownPageProps = {
+    /** Value of the policy report field */
+    fieldValue: string;
+
+    /** Name of the policy report field */
+    fieldName: string;
+
+    /** Options of the policy report field */
+    fieldOptions: string[];
+
+    /** Callback to fire when the Save button is pressed  */
+    onSubmit: () => void;
+};
+
+function EditReportFieldDropdownPage({fieldName, onSubmit, fieldValue, fieldOptions}: EditReportFieldDropdownPageProps) {
+    const [searchValue, setSearchValue] = useState('');
+    const styles = useThemeStyles();
+    const {getSafeAreaMargins} = useStyleUtils();
+    const {translate} = useLocalize();
+
+    const sections = useMemo(() => {
+        const filteredOptions = fieldOptions.filter((option) => option.includes(searchValue));
```

Comment:
```suggestion
        const filteredOptions = fieldOptions.filter((option) => option.toLowerCase().includes(searchValue.toLowerCase()));
```
I guess we'd want case-insensitive matching here.